examples/grayscale_rgba_example.html

```diff
@@ -0,0 +1,47 @@
+<html>
+
+<body>
+    <img id="img" src="pinball.jpg">
+    <canvas id="canvas" width="1637" height="2048"></canvas>
+    <canvas id="dst" width="1637" height="2048"></canvas>
+
+    <script type="module">
+        import jsfeatCpp from "./../build/jsfeatES6cpp_debug.js"
+        const jsfeat = await jsfeatCpp();
+        console.log(jsfeat);
+
+        var canvas = document.getElementById('canvas');
+        var ctx = canvas.getContext('2d');
+        var canvasDst = document.getElementById('dst');
+        var ctxDst = canvasDst.getContext('2d');
+        var img = document.getElementById('img');
+        ctx.drawImage(img, 0, 0);
+        var data = ctx.getImageData(0, 0, img.width, img.height);
+
+        const RGBA2GRAY = jsfeat.Colors.COLOR_RGBA2GRAY.value;
+        const U8_t = jsfeat.Types.U8_t.value;
+        const C1_t = jsfeat.Types.C1_t.value;
+
+        var m_u8 = new jsfeat.Matrix_smart(img.width, img.height, U8_t | C1_t);
+        console.log(m_u8);
+        var imgproc = new jsfeat.imgproc();
+
+        if (data) {
+            imgproc.grayscale_rgba(data.data, img.width, img.height, m_u8.getSmartPointer());
+            
+            const imageData = ctxDst.createImageData(img.width, img.height);
+            var inputData = m_u8.data;
+            m_u8.delete()
+            console.log(inputData);
```

Comment:
the output is sequence of zeros, instead the grayimage is displayed. Why this?

Reply:
My affirmationis not correct, there aren't only zeros, infact after 6000000 yiou will some values and that's why the image is displayed. But because it is shifted you will see at the beginning only zeros. 
differences between grayscale and grayscale_rgba:
grayscale_rgba Uint8Array is `6705152` in length
grayscale Uint8Array is `3352576` in length